lib/store/offline_authenticator_db.dart

```diff
@@ -25,8 +26,20 @@ class OfflineAuthenticatorDB {
   }
 
   Future<Database> _initDatabase() async {
+    if (Platform.isWindows || Platform.isLinux) {
+      var databaseFactory = databaseFactoryFfi;
+      return await databaseFactory.openDatabase(
+        (await getDownloadsDirectory())!
```

Comment:
nit: we could extract this logic of getting the database directory to a common place